covariance/covarianceOsc.h

```diff
@@ -23,7 +20,15 @@ class covarianceOsc : public covarianceBase
       void setFlipBeta(bool flip){flipBeta=flip;}
       void useReactorPrior(bool reactor){reactorPrior = reactor;};
       void setExtraBranches(TTree &tree);
-      
+
+      double GetPathLength() {
+        return L;
+      }
+
+      double GetDensity() {
+        return density;
+      }
```

Comment:
In long term we should make them proper syst parameters. In short term we can inline ;)